delfin/drivers/dell_emc/vmax/vmax.py

```diff
@@ -46,10 +48,13 @@ def get_storage(self, context):
         free_cap = total_cap - used_cap
 
         storage = {
-            'name': '',
+            # Unisphere Rest API do not provide Array name .
+            # Generate  name  by combining model and symmetrixId
+            'name': model + '-' + self.client.array_id,
```

Comment:
why not display_name?

Reply:
as discussed, no display name in rest reponse.

Reply:
Thanks, this is available only in {version}/system path. Will change the URL to fech system name..

Reply:
thanks, modified to display_name